src/components/AppointmentConfigModal.vue

```diff
@@ -206,6 +218,12 @@ export default {
 			const uri = this.editing.targetCalendarUri
 			return this.ownSortedCalendars.find(cal => this.calendarUrlToUri(cal.url) === uri)
 		},
+		conflictCalendars() {
+			const freebusyUris = this.editing.freebusyUris ?? []
```

Comment:
unused?

Reply:
It's used in the next line.

Reply:
It's used in the next line.

Reply:
right … got confused by the different colors here on the github diff …